<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PLZ_WORK</string>
  </property>
  <property name="toolTipDuration">
   <number>-1</number>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="b_loadData">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>100</y>
      <width>500</width>
      <height>60</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string>Загрузить данные из файлов</string>
    </property>
   </widget>
   <widget class="QPushButton" name="b_addPart">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>170</y>
      <width>500</width>
      <height>60</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string>Регистрация поступления комплектующих</string>
    </property>
   </widget>
   <widget class="QPushButton" name="b_viewAll">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>240</y>
      <width>501</width>
      <height>60</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string>Посмотреть все остатки</string>
    </property>
   </widget>
   <widget class="QPushButton" name="b_createRequest">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>310</y>
      <width>241</width>
      <height>60</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string>Создать заявку на поставку</string>
    </property>
   </widget>
   <widget class="QPushButton" name="b_viewAKB">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>380</y>
      <width>500</width>
      <height>60</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string>Посмотреть остатки АКБ</string>
    </property>
   </widget>
   <widget class="QPushButton" name="b_viewRequests">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>310</y>
      <width>231</width>
      <height>60</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string>Посмотреть заявки</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
